client/src/SimilarProductsComponent.jsx

```diff
@@ -32,12 +32,32 @@ const SliderButton = styled.button`
   font-size: 1.2rem;
   align-self: center;
   position: absolute;
+  left: 10px;
   z-index: 10;
   box-shadow: 0px 10px 15px #888888;
   height: 50px;
   width: 50px;
 `;
 
+const RightButton = styled.button`
```

Comment:
You can also copy the LeftButton and override a few of the styles that needs changing. Like for example, if the RightButton needs to have different color than the LeftButton but everything else is same, you can do:
const RightButton = styled(LeftButton)`
  background-color: green;
`
Just a good way to make your code "DRY"